Scripts/Items/Containers/BaseDungeonChest.cs

```diff
@@ -1,21 +1,17 @@
-// Treasure Chest Pack - Version 0.99H
-// By Nerun
-
 using System;
 
 namespace Server.Items
 {
-    public abstract class BaseTreasureChestMod : LockableContainer
+    public abstract class BaseDungeonChest : LockableContainer
```

Comment:
Does that use resources though? Like if we can avoid it, should we? Since these are just an auto spawn, not even accurate implementation of "dungeon chests" that ServUO has always had, figured just one out of cycle restart is better.

I get it for actual items players collect or can pick up.

Reply:
No. Its a simple check during deserializion when you change the name of a class. W/out using it, all those items are delteed.

Reply:
Which is fine though, they will respawn.